lib/cc/engine/analyzers/reporter.rb

```diff
@@ -93,8 +133,13 @@ def flay_options
           CCFlay.default_options.merge changes
         end
 
-        def debug(message)
-          $stderr.puts(message) if engine_config.debug?
+        require "thread"
+        IO_M = Mutex.new
```

Comment:
Can you move this constant declaration to the top of the class above `def initialize`?